geonode/api/resourcebase_api.py

```diff
@@ -435,17 +441,57 @@ def get_list(self, request, **kwargs):
             to_be_serialized)
         return self.create_response(request, to_be_serialized)
 
-    def create_response(
-            self,
-            request,
-            data,
-            response_class=HttpResponse,
-            **response_kwargs):
-        """
-        Extracts the common "which-format/serialize/return-response" cycle.
+    def apply_perm_sorting(self, request, objs):
 
-        Mostly a useful shortcut/hook.
-        """
+        permitted_view_id = get_objects_for_user(
+            request.user,
+            'base.view_resourcebase').values_list(
+            'id',
+            flat=True)
+
+        permited_change_perms = get_objects_for_user(
+            request.user,
+            'base.change_resourcebase_permissions').values_list(
+            'id',
+            flat=True)
+
+        obj_no_perms = []
+        obj_perms = []
+
+        # retrieving in the json response the permisions
+        # to view resource 'can_view_resource'
+        # to change resource perms 'can_change_perms'
+        for idx, obj in enumerate(objs):
```

Comment:
I did not write this part of the code, so I will give you an answer asap. 
Does this can help you : [http://lists.osgeo.org/pipermail/geonode-users/2015-June/000519.html](url) ? 
Thanks